README.md

```diff
@@ -73,15 +218,27 @@ The following is a non-normative example of a W3C VC using the internal proof fo
   "id": "https://example.com/credentials/1872",
   "type": [
     "VerifiableCredential",
-    "AlumniCredential"
+    "IDCardredential"
   ],
   "issuer": {
     "id": "did:example:issuer"
   },
   "issuanceDate": "2010-01-01T19:23:24Z",
   "credentialSubject": {
-    "id": "did:example:holder",
-    "alumniOf": "Example University"
+    "given_name": "Fredrik",
+    "family_name": "Strömberg",
+    "birthdate": "1949-01-22",
+    "place_of_birth": {
+       "country": "SE",
+       "locality": "Örnsköldsvik"
+    },
+     "nationality": "SE",
+    "number": "4901224131",
+    "date_of_issuance":"2010-03-23",
+    "date_of_expiry":"2020-03-22"
+    "issuer":{
```

Comment:
@tlodderstedt the `issuer` property should be one level above. In that case likely merged with the `issuer` property.

Reply:
done

Reply:
it seems to make sense here.